### DISCUSSION TITLE
To Open in Windows 11

### DISCUSSION THREAD

**Discussion by scottytoohotty3** [elementary/triage#412] [Ideas]
Hi, is it possible to make a download for elementry os to load into elementry os straight from windows os. Would like to see this option in future. thank you and have a nice day.

**Comment by teamcons** [comment]
you'll need to be more explicit.

**Comment by scottytoohotty3** [comment]
Hi, make an elementryos.exe for windows to download. Have it install so you can auto-run or load directly into elementry OS from microsoft windows os. Thank you. Have a nice day.
